# relative pos of the base_link w.r.t shelf_frame expressed in shelf_frame
# format: [x y z roll pitch yaw] in meters and radians

base_pos_dict: 
  column_1: [-0.82, 0.02, 0.0, 0.0, 0.0, 0.0]
  column_2: [-0.82, -0.2705, 0.0, 0.0, 0.0, 0.0]
  column_3: [-0.82, -0.54, 0.0, 0.0, 0.0, 0.0]
      
  drop:
    move_right: [-1.33, -0.5, 0.0, 0.0, 0.0, 0.0]

    retreat_cheezit_big_original: [-1.571, -0.5, 0.0, 0.0, 0.0, 0.0]
    retreat_oreo_mega_stuf: [-1.321, -0.5, 0.0, 0.0, 0.0, 0.0]
    retreat: [-1.451, -0.5, 0.0, 0.0, 0.0, 0.0]

    # y coordinate when moving left before dropping
    move_left_y: -0.1275
 
 